sopel/irc/backends.py

```diff
@@ -278,10 +283,14 @@ def handle_connect(self):
         # version(s) it supports.
         if not self.verify_ssl:
             self.ssl = ssl.wrap_socket(self.socket,  # lgtm [py/insecure-default-protocol]
```

Comment:
I feel like it's time to fix that LGTM issue while we are playing with certfile.

Reply:
That might be better with other changes (dropping `ssl.wrap_socket()`, etc) that'd make a new PR.

Reply:
I did notice the opportunity for switching from `ssl.wrap_socket()` to using an `SSLContext()`. Agree that such a refactoring should come separately (and not just because I don't feel like doing it right now 😝).